### PR TITLE
Improvement: Smart market config

### PR DESCRIPTION
**Pull request by mgufrone**
- [x] Enable the markets and currencies goroutine fetch only when sonic.private.xln_enabled == true
- [x] Market configuration update

### CODE REVIEW

File: skel/daemons/license.go [hunk 1]
```diff
@@ -242,3 +258,45 @@ func saveLicenseToVault(app string, vaultService *vault.Service, license string)
...
 }
+func getFinexRestart(vaultService *vault.Service) (int64, error) {
+	app := "finex"
+	scope := "private"
+
+	// Load secret
+	vaultService.LoadSecrets(app, scope)
+
+	// Get secret
+	licRaw, err := vaultService.GetSecret(app, "finex_restart", scope)
```
Comment (vshatravenko) [inline skel/daemons/license.go:269]: licRaw means licenseRaw, I think you should rename this one :)

File: skel/daemons/license.go [hunk 2]
```diff
@@ -185,6 +185,22 @@ func getPlatformIDFromVault(vaultService *vault.Service) (string, error) {
 	return result.(string), nil
 }
 
+func GetXLNEnabledFromVault(vaultService *vault.Service) (bool, error) {
```
Comment (vshatravenko) [inline skel/daemons/license.go:188]: Let's move this into markets.go, it's not related to the license

File: skel/daemons/license.go [hunk 3]
```diff
@@ -242,3 +258,45 @@ func saveLicenseToVault(app string, vaultService *vault.Service, license string)
 
 	return nil
 }
+func getFinexRestart(vaultService *vault.Service) (int64, error) {
```
Comment (vshatravenko) [inline skel/daemons/license.go:261]: Let's move this to markets.go

File: skel/handlers/index.go [hunk 4]
```diff
@@ -106,7 +106,13 @@ func Setup(app *sonic.Runtime) {
 	go daemons.LicenseRenewal("finex", app, vaultService)
 
 	// Run FetchMarkets
-	go daemons.FetchMarkets(peatioClient, vaultService, opendaxConfig.Addr)
+	enabled, err := daemons.GetXLNEnabledFromVault(vaultService)
+	if err != nil {
+		log.Printf("Can't create peatio client: " + err.Error())
```
Comment (vshatravenko) [inline skel/handlers/index.go:111]: I'm not sure about this error message, should be something about getting the xln_enabled value

**Review by vshatravenko — CHANGES_REQUESTED** [review]

File: skel/handlers/index.go (hunk 4 above)
Comment (mgufrone) [reply skel/handlers/index.go:111]: @vshatravenko what would you suggest for this? should we just ignore the error message? I mean, we're sort of just using the boolean value, while by default it always `false` when error occured.

Comment (vshatravenko) [reply skel/handlers/index.go:111]: We need the error message, just it shouldn't mention peatio since it's about XLN

**Review by vshatravenko — APPROVED** [review]

**Pull request merged**